bin/transform.py

```diff
@@ -274,5 +337,16 @@ def main():
         metadata_readable.to_csv(RESULTS_PATH, index=False)
         metadata_irida.to_csv(TRANSFORMATION_PATH, index=False)
 
+    elif (args.transformation == CATEGORIZE):
+
+        metadata_readable, metadata_irida = categorize(metadata)
+
+        remove_all_NA_columns(metadata_irida)
+        remove_all_NA_columns(metadata_readable)
```

Comment:
Unless I'm wrong, I don't think we want to remove empty/NA columns from `metadata_readable`. I think the "full", readable dataset should have empty rows/columns so that it helps debug.

They're removed from `metadata_irida` because irida will write back and overwrite empty values, which we usually don't want to do.